libsolidity/inlineasm/AsmCodeGen.h

```diff
@@ -42,8 +42,8 @@ struct Block;
 class CodeGenerator
 {
 public:
-	CodeGenerator(ErrorList& _errors):
-		m_errors(_errors) {}
+	CodeGenerator(ErrorReporter& _errorReporter):
```

Comment:
And, actually, because it includes `AsmAnalysis.h`, it could get by with nothing at all, but the forward declaration is good for self-documenting code.